src/components/AssetMeta/components/AssetEventsPanel.test.tsx

```diff
@@ -12,13 +12,35 @@ describe('AssetEventsPanel', () => {
     const tree = renderer.create(<AssetEventsPanel events={EVENTS} />).toJSON();
     expect(tree).toMatchSnapshot();
   });
-  it('checks if the modal is opened', () => {
+  it('checks if the modal is opened', done => {
     const EventPanelModal = mount(<AssetEventsPanel events={EVENTS} />);
 
     expect(EventPanelModal.state('selectedEvent')).toEqual(null);
 
     const clickableRow = EventPanelModal.find('td').first();
     clickableRow.simulate('click');
-    expect(typeof EventPanelModal.state('selectedEvent')).toBe('object');
+    setImmediate(() => {
+      EventPanelModal.update();
+      const modalWindow = EventPanelModal.find('Modal');
+      expect(modalWindow).toHaveLength(1);
+      done();
+    });
   });
+  it.each`
+    inlineStyle                              | expected
+    ${'style="width: 80%;"'}                 | ${true}
+    ${'style="background: rgb(0, 255, 0);"'} | ${true}
+    ${'style="font-style: italic;"'}         | ${true}
+  `(
+    'should render component with passed inline styles',
+    ({ inlineStyle, expected }) => {
+      const EventPanel = mount(
+        <AssetEventsPanel events={EVENTS} styles={ASSET_META_STYLES.events} />
+      );
+      const containerTable = EventPanel.find('.ant-table-wrapper')
+        .last()
+        .html();
```

Comment:
It's not a good way to check if styles passing to expected elements. Suggestion – use `.prop('style')` on expected element to get `style` value and compare with passed to element.

Reply:
Test redesigned